src/config/generateCode.ts

```diff
@@ -0,0 +1,9 @@
+export const generateCode = () => {
```

Comment:
This should rather be in a service. Let all business logic stay in service please